workshops/python and data visualizations/README.md

```diff
@@ -0,0 +1,325 @@
+![Stonks](https://wompampsupport.azureedge.net/fetchimage?siteId=7575&v=2&jpgQuality=100&width=700&url=https%3A%2F%2Fi.kym-cdn.com%2Fentries%2Ficons%2Ffacebook%2F000%2F029%2F959%2FScreen_Shot_2019-06-05_at_1.26.32_PM.jpg)
+
+You after you make your first data visualization^
```

Comment:
Please add the metadata as requested in the bounty guidelines to the start of the file